### ISSUE TITLE
Container pb33f/openapi-changes:latest missing GLIBC_2.32, GLIBC_2.34

### ISSUE THREAD

**Issue by tillig** [pb33f/openapi-changes#45]
```text
> docker run --rm pb33f/openapi-changes --help
openapi-changes: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by openapi-changes)
openapi-changes: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by openapi-changes)
```

This specific `latest` is `sha256:cd5c3d32dd8c11c4b0529afe5ddf2ccc41c15e7ebae2c79819dfcbc6b9e97634`. It appears there have been some upgrades to the Dockerfile and build with respect to the Go and dependency versions, I'm guessing that's what brought this on.

**Comment by tillig** [comment]
It appears only v0.0.27 and v0.0.28 (latest) containers are available and both are affected. I would try an older one but there are no older tags.

**Comment by daveshanley** [comment]
Yes, I am having some trouble with nixpacks and the version of go they are using is out of date. I will revert the change as

**Comment by daveshanley** [comment]
Should, be resolved now in the latest tag as well as `v0.0.30`

**Comment by tillig** [comment]
Verified, things seem to be back up and running. Thanks!
